musl/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:edge
+
+# TODO: change to container registry
+COPY --from=clangbuiltlinux/containers:llvm-project llvm-project/llvm/build/bin /usr/local/bin
+COPY --from=clangbuiltlinux/containers:llvm-project /usr/local/lib /usr/local/lib
+COPY --from=clangbuiltlinux/containers:llvm-project /usr/local/include /usr/local/include
```

Comment:
TODO: I need to update this to drop `containers` and use `clangbuiltlinux/llvm-project:latest`.